sycl/test/fpga_tests/fpga_aocx.cpp

```diff
@@ -9,7 +9,9 @@
 // REQUIRES: aoc, accelerator
 
 /// E2E test for AOCX creation/use/run for FPGA
-// Produce an archive with device (AOCX) image
+// Produce an archive with device (AOCX) image. Pre-create a file to avoid
+// appending objects to leftover archives.
+// RUN: touch %t_image.a
```

Comment:
I would assume that the `%t` is going to be the same with each consecutive run.  I don't think that just touching the file will prevent the appending to the image we probably should remove instead.

Reply:
Agree. touch will only change modify  time if file present. Please do the remove.

Reply:
Thanks, I've made adjustments to see if this helps